noir/noir-repo/tooling/noirc_abi_wasm/test/shared/abi_encode.ts

```diff
@@ -9,9 +9,7 @@ export const abi: Abi = {
       visibility: 'private',
     },
   ],
-  param_witnesses: { foo: [{ start: 1, end: 2 }], bar: [{ start: 2, end: 4 }] },
```

Comment:
These had to be removed after https://github.com/noir-lang/noir/pull/5154. This data structure mismatch wasn't caught in Noir's CI @TomAFrench